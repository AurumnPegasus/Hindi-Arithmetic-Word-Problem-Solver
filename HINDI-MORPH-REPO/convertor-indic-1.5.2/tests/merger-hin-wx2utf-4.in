<Sentence id='4'>
1	((	NP	
1.1	((	NP	<fs etype="timex" esubtype="date" value="19:06:1930"  >
1.1.1	jUna	NNP	<fs af='jUna,n,m,sg,3,d,0,0' esubtype="date" etype="timex" value="xx:06:xx"  >
1.1.2	((	NP	<fs etype="numx" value="19.0"  >
1.1.2.1	19	QC	<fs af='19,num,,,,,,' etype="numx" value="19.0"  >
	))
1.1.3	,	SYM	<fs af='&comma,punc,,,,,,'  >
1.1.4	((	NP	<fs etype="numx" value="1930.0"  >
1.1.4.1	1930	NN	<fs af='1930,num,,,,,,' etype="numx" value="1930.0"  >
	))
	))
	))
2	((	NP	
2.1	ke	PSP	<fs af='kA,psp,m,sg,,o,kA,kA'  >
	))
3	((	NP	
3.1	((	NP	<fs etype="timex" esubtype="dayOfWeek"  >
3.1.1	maMgalavAra	NNP	<fs af='maMgalavAra,n,m,sg,3,d,0,0' esubtype="dayOfWeek" etype="timex"  >
	))
3.2	kI	PSP	<fs af='kA,psp,f,sg,,d,kA,kA'  >
	))
4	((	NP	
4.1	((	NP	<fs etype="timex" esubtype="timeOfDay"  >
4.1.1	subaha	NN	<fs af='subaha,n,f,sg,3,d,0,0' esubtype="timeOfDay" etype="timex"  >
	))
	))
5	((	NP	
5.1	yuxXa	NN	<fs af='yuxXa,n,m,sg,3,o,0,0'  >
5.2	ko	PSP	<fs af='ko,psp,,,,,,'  >
	))
6	((	NP	
6.1	((	NP	<fs etype="timex" esubtype="timeMeasure" unit="Month" value="10.0"  >
6.1.1	((	NP	<fs etype="numx" value="10.0"  >
6.1.1.1	xasa	QC	<fs af='xasa,adj,any,pl,,d,,' etype="numx" value="10.0"  >
	))
6.1.2	mahIne	NN	<fs af='mahInA,n,m,pl,3,d,0,0' esubtype="timeMeasure" etype="timex" value="1.0" Unit="Month"  >
	))
	))
7	((	JJP	
7.1	pUre	JJ	<fs af='pUrA,adj,m,pl,,d,,'  >
	))
8	((	VGF	
8.1	hoMge	VM	<fs af='ho,v,m,pl,3,,gA,gA'  >
8.2	.	SYM	<fs af='.,punc,,,,,,'  >
	))
</Sentence>

