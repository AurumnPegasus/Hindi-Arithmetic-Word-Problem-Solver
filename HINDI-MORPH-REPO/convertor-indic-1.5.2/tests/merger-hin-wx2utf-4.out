<Sentence id='4'>			
1	((	NP	
1.1	((	NP	<fs  esubtype="date"  etype="timex"  value="19:06:1930">
1.1.1	जून	NNP	<fs af='जून,n,m,sg,3,d,0,0' esubtype="date"  etype="timex"  value="xx:06:xx">
1.1.2	((	NP	<fs  etype="numx"  value="19.0">
1.1.2.1	19	QC	<fs af='19,num,,,,,,' etype="numx"  value="19.0">
	))
1.1.3	,	SYM	<fs af='&comma,punc,,,,,,'>
1.1.4	((	NP	<fs  etype="numx"  value="1930.0">
1.1.4.1	1930	NN	<fs af='1930,num,,,,,,' etype="numx"  value="1930.0">
	))
	))
	))
2	((	NP	
2.1	के	PSP	<fs af='का,psp,m,sg,,o,का,kA'>
	))
3	((	NP	
3.1	((	NP	<fs  esubtype="dayOfWeek"  etype="timex">
3.1.1	मंगलवार	NNP	<fs af='मंगलवार,n,m,sg,3,d,0,0' esubtype="dayOfWeek"  etype="timex">
	))
3.2	की	PSP	<fs af='का,psp,f,sg,,d,का,kA'>
	))
4	((	NP	
4.1	((	NP	<fs  esubtype="timeOfDay"  etype="timex">
4.1.1	सुबह	NN	<fs af='सुबह,n,f,sg,3,d,0,0' esubtype="timeOfDay"  etype="timex">
	))
	))
5	((	NP	
5.1	युद्ध	NN	<fs af='युद्ध,n,m,sg,3,o,0,0'>
5.2	को	PSP	<fs af='को,psp,,,,,,'>
	))
6	((	NP	
6.1	((	NP	<fs  esubtype="timeMeasure"  etype="timex"  value="10.0"  unit="Month">
6.1.1	((	NP	<fs  etype="numx"  value="10.0">
6.1.1.1	दस	QC	<fs af='दस,adj,any,pl,,d,,' etype="numx"  value="10.0">
	))
6.1.2	महीने	NN	<fs af='महीना,n,m,pl,3,d,0,0' esubtype="timeMeasure"  etype="timex"  value="1.0"  Unit="Month">
	))
	))
7	((	JJP	
7.1	पूरे	JJ	<fs af='पूरा,adj,m,pl,,d,,'>
	))
8	((	VGF	
8.1	होंगे	VM	<fs af='हो,v,m,pl,3,,गा,gA'>
8.2	.	SYM	<fs af='.,punc,,,,,,'>
	))
</Sentence>			
			
